tests/datadriven/test_variants.py

```diff
@@ -56,11 +57,77 @@ def getDataModelClass(self):
     def getProtocolClass(self):
         return protocol.GAVariantSet
 
-    def verifyVariantsEqual(self, gaVariants, pyvcfVariants):
+    def _compareTwoFloats(self, a, b):
+        afloat = float(a)
+        bfloat = float(b)
+        return (abs(afloat - bfloat) <= 0.001 * abs(bfloat))
+
+    def _compareTwoListFloats(self, a, b):
+        for ai, bi in zip(a, b):
+            if not self._compareTwoFloats(ai, bi):
+                return False
+        return True
+
+    def _verifyInfoEqual(self, gaObjectInfo, pyvcfInfo):
+        def _compareGaValuePyvcfValue():
```

Comment:
it's an assertion, not a comparison. And I think I'd like to see values passed in explicitly, makes for less error prone code if somebody changes the variable names later in the code:
```
    def _assertEquivalentGaVCFValues(ga, pyvcf):
[...]
    #usage:
    _assertEquivalentGaVCFValues(gaValue, pyvcfValue)
```